atmo/models.py

```diff
@@ -13,6 +13,28 @@
 from guardian.utils import get_user_obj_perms_model
 
 
+class RegexpReplace(models.Func):
+    function = 'REGEXP_REPLACE'
+
+    def __init__(self, expression, pattern, replacement, flags=None, **extra):
+        if not hasattr(pattern, 'resolve_expression'):
+            if not isinstance(pattern, str):
+                raise TypeError("'pattern' must be a string")
+            pattern = models.Value(pattern)
+        if not hasattr(replacement, 'resolve_expression'):
+            if not isinstance(replacement, str):
+                raise TypeError("'replacement' must be a string")
+            replacement = models.Value(replacement)
+        expressions = [expression, pattern, replacement]
+        if flags is not None:
+            if not hasattr(flags, 'resolve_expression'):
+                if not isinstance(flags, str):
+                    raise TypeError("'flags' must be a string")
+                flags = models.Value(flags)
+            expressions.append(flags)
+        super().__init__(*expressions, **extra)
```

Comment:
Any way to test this?

Reply:
Shrug, I guess, seems wasted time though.